src/ui-kit/SliderYears/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { getEndYear, getStartYear } from 'redux/selectors'
+import { setCurrentYearsDate, setCurrentYearsAfter } from 'redux/actions'
+
+import { number } from 'prop-types'
+
+import { Slider } from 'antd'
```

Comment:
correct these imports please..........................................................................................

Reply:
we discussed this thing a million times :) 

1. imports from node_modules (import shit from 'react')
2. absolute imports (import shit from 'common/components')
3. relative imports (../../../../)
4. styles import (import './styles.scss')